ArcGISRuntimeSDKQt_CppSamples/Layers/PlayAKmlTour/README.md

```diff
@@ -0,0 +1,51 @@
+# Play a KML Tour
+
+Play tours in KML files.
+
+![](screenshot.png)
+
+## Use case
+
+KML, the file format used by Google Earth, supports creating tours, which can control the viewpoint of the scene, hide and show content, and play audio. Tours allow you to easily share tours of geographic locations, which can be augmented with rich multimedia. Runtime allows you to consume these tours using a simple API.
+
+## How to use the sample
+
+The sample will load the KMZ file automatically. When a tour is found, the _Play_ button will be enabled. Use _Play_ and _Pause_ to control the tour. When you're ready to show the tour, use the reset button to return the tour to the unplayed state.
+
+## How it works
+
+1. Load the KML dataset and add it to a layer.
+2. Create the KML tour controller. Wire up the buttons to the `play()`, `pause()`, and `reset()` methods.
+3. Use a method to explore the tree of KML content and find the first KML tour. Once a tour is found, provide it to the KML tour controller.
+4. Enable the buttons to allow the user to play, pause, and reset the tour.
+
+## Relevant API
+
+* KmlTourController
+* KmlTourController.tour
+* KmlTourController.play()
```

Comment:
In the C++ readmes we usually try to use :: for relevant API - KmlTourController::tour